ember-file-upload/package.json

```diff
@@ -99,12 +98,18 @@
     "release-it-lerna-changelog": "^5.0.0",
     "rollup": "^2.70.1",
     "rollup-plugin-ts": "^3.0.1",
+    "tracked-built-ins": "^3.0.0",
     "typescript": "^4.7.2",
     "webpack": "^5.74.0"
   },
   "peerDependencies": {
+    "@ember/test-helpers": "^2.9.3",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
     "ember-cli-mirage": "*",
-    "miragejs": "*"
+    "ember-modifier": "^3.2.7 || ^4.1.0",
+    "miragejs": "*",
+    "tracked-built-ins": "^3.1.1"
   },
```

Comment:
Everything I've moved to `peerDependencies` is in the modern `ember-cli` blueprints.